src/components/locationForm/components/TextAreaInput.jsx

```diff
@@ -16,42 +16,23 @@ const StyledInput = styled(Input)`
     `}
 `;
 
-const TextAreaInput = ({
-  name,
-  label,
-  defaultValue,
-  onChange,
-  ...otherProps
-}) => {
-  const [value, setValue] = useState('');
-
-  useEffect(() => {
-    setValue(defaultValue || '');
-  }, [defaultValue]);
-
-  const onValueChange = e => {
-    setValue(e.target.value);
-    onChange(e);
-  };
-
+const TextAreaInput = ({ name, value, onChange, ...otherProps }) => {
   return (
     <StyledInput
       as="textarea"
       name={name}
-      value={value}
+      value={value || ''}
```

Comment:
Why have an empty string here? The prop `value` is a required prop.

Reply:
yes, this helps prevent assigning of the null values.